app/assets/javascripts/application.js

```diff
@@ -69,7 +69,7 @@ function onContentLoad(){
 
   //set the tooltips
   $('a[rel="popover"]').popover({html: true});
-  $('[rel="twipsy"]').tooltip();
+  $('[rel="twipsy"]').tooltip({ container: 'body' });
```

Comment:
This was added since tooltips inside the org/location dropdowns where limited to the size of the dropdown and couldn't show the full title.
The problem is it affects all tooltips so better solutions are welcome.